lib/service/servicecfg/auth.go

```diff
@@ -253,6 +255,10 @@ func (cfg *PKCS11Config) CheckAndSetDefaults() error {
 	if cfg.SlotNumber == nil && cfg.TokenLabel == "" {
 		return trace.BadParameter("must provide one of SlotNumber or TokenLabel")
 	}
+
+	if cfg.MaxSessions == 1 {
```

Comment:
```suggestion
	if cfg.MaxSessions <= 1 {
```

It's a signed integer, so might as well protect against 0 or negative values too.